src/util/belt.rs

```diff
@@ -105,29 +110,23 @@ impl StagingBelt {
             .get_mapped_range_mut()
     }
 
-    /// Produce a command buffer with all the accumulated transfers.
+    /// Prepare for command buffers to be submitted.
     ///
     /// At this point, all the partially used staging buffers are closed until
     /// the GPU is done copying the data from them.
-    pub fn flush(&mut self, device: &Device) -> CommandBuffer {
+    pub fn flush(&mut self) {
```

Comment:
I think with the new signature it should no longer be called `flush`. Maybe `finish()?`